poetry.lock

```diff
@@ -758,13 +758,13 @@ colors = ["colorama (>=0.4.6)"]
 
 [[package]]
 name = "jinja2"
-version = "3.1.3"
+version = "3.1.4"
 description = "A very fast and expressive template engine."
-optional = true
+optional = false
```

Comment:
This is incorrect. According to poetry.lock, the only package here that uses jinja2 is Sphinx. Sphinx is optional, so Sphinx's dependencies should be optional unless a required package also needs them.